_translator-files/po/de/Include-Server-Commands.po

```diff
@@ -3,128 +3,129 @@
 # This file is distributed under the same license as the PACKAGE package.
 # FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
 #
-#, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
-"Language: \n"
+"Project-Id-Version: \n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: 2022-05-16 17:41+0200\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: de\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 3.0.1\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Server-Commands.md:2
 #, no-wrap
 msgid "`-d`  or `--discononquit`   Disconnect all Clients on quit. Normally, when a Server is stopped or restarted, any Clients that have not used their \"Disconnect\" buttons will re-establish connection when the Server comes back up again. Using this option forces Clients to manually re-establish their connections to the Server.  \n"
-msgstr ""
+msgstr "`-d` oder `--discononquit` Trennt alle Clients beim Beenden. Wenn ein Server angehalten oder neu gestartet wird, werden normalerweise alle Clients, die ihre \"Disconnect\"-Schaltflächen nicht benutzt haben, die Verbindung wiederherstellen, wenn der Server wieder hochfährt. Mit dieser Option werden die Clients gezwungen, ihre Verbindungen zum Server manuell wiederherzustellen.\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Server-Commands.md:3
 #, no-wrap
 msgid "`-e`  or `--directoryserver`  Register the Server on a Directory (e.g. to set its genre (see also `-o`)). See [Server Types](#server-types) for further information.\n"
-msgstr ""
+msgstr "`-e` oder `--directoryserver` Registriert den Server in einem Verzeichnis (z.B. um sein Genre zu setzen (siehe auch `-o`)). Siehe [Server-Typen](#server-typen) für weitere Informationen.\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Server-Commands.md:4
 #, no-wrap
 msgid "`--directoryfile` Remember registered Servers even if the Directory is restarted. Directory Servers only. See [this guide](Custom-Directories) for further information. \n"
-msgstr ""
+msgstr "`--directoryfile` Registrierte Server speichern, auch wenn das Verzeichnis neu gestartet wird. Nur für Verzeichnis-Server. Siehe [dieses Handbuch](Benutzerdefinierte Verzeichnisse) für weitere Informationen\n"
```

Comment:
Please check the link - as above.

Reply:
Corrected
Siehe [Informationen zum Verzeichnis Server](Running-a-Server#3-verzeichnis-server)